test_scripts/Polices/build_options/ATF_P_TC_PTU_Cycleing_Through_The_URLs_During_Retry_Sequence.lua

```diff
@@ -135,4 +135,10 @@ function Test:ValidateResult()
   end
 end
 
+--[[ Postconditions ]]
```

Comment:
@istoimenova, Please correct in line 133: 
FailTestCase() function can be invoked just only once in Test() function 
Otherwise next Test() functions will be failed --> this this some restriction of ATF